rtv/content.py

```diff
@@ -162,9 +166,14 @@ def strip_praw_submission(sub):
         data['comments'] = '{} comments'.format(sub.num_comments)
         data['score'] = '{} pts'.format(sub.score)
         data['author'] = (clean(sub.author.name) if getattr(sub, 'author')
+
                           else '[deleted]')
         data['permalink'] = clean(sub.permalink)
         data['subreddit'] = strip_subreddit_url(sub.permalink)
+        if sub.link_flair_text != None:
```

Comment:
You might be able to get away with a [ternary](https://docs.python.org/3.3/faq/programming.html#is-there-an-equivalent-of-c-s-ternary-operator) for this, and the one in the strip_praw_comment()
```python
data['flair'] = (clean(sub.link_flair_text) if sub.link_flair_text else "")
```
IMO it's a bit cleaner and matches the rest of the code style